setup/paths.sh

```diff
@@ -37,38 +37,27 @@ else
     source $BUILD_HOME/cmsgemos/setup/paths.sh
 fi
 
+# Export project
+export GEM_PLOTTING_PROJECT=$BUILD_HOME/gem-plotting-tools
```

Comment:
In `cmsgemos` it is used mainly in the RPM building and `make` processes.
What code in `vfatqc-python-scripts` actually uses this variable, or depends on it being set (or is it simply a way to see if the software is present)?